front_end/panels/ai_chat/LLM/GenericOpenAIProvider.ts

```
@@ -0,0 +1,447 @@
+
+      return data;
+    } catch (error) {
+      logger.error(`${this.displayName} API request failed:`, error);
+      throw error;
+    }
+  }
+
+  /**
+   * Processes the API response and converts to LLMResponse format
+   */
+  private processOpenAIResponse(data: any): LLMResponse {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)